cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManager.java

```diff
@@ -590,6 +593,48 @@ boolean isUserTaskDoneExceptionally() {
     }
   }
 
+  // Persist user request and result to log. Called when the user task completes and is moved into COMPLETED status
+  private void persistUserTask(UserTaskInfo userTaskInfo) {
```

Comment:
This function should be called by a separate thread. We should not be blocking when this expensive method is called.